handbook_wip/README.md

```diff
@@ -0,0 +1,9 @@
+# Handbook WIP
```

Comment:
I'm afraid that we need to maintain 2 READMEs now when we change a file

Reply:
I think it's a nice way to introduce the handbook as a concept. I don't consider it a README we'll need to update regularly.

Reply:
We are able to see the same structure just above the README. 
I will delete this and get it merged. We can talk about it outside the PR.